backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs

```diff
@@ -272,7 +272,9 @@ public async Task DeleteAvailabilitySlot(long slotId)
         {
             var slot = await _context.AvailabilitySlots.FirstAsync(el => el.Id == slotId);
             var members = _context.SlotMembers.Where(member => member.SlotId == slotId);
+            var meetings = _context.Meetings.Where(_ => _.AvailabilitySlotId == slotId);
```

Comment:
maybe make normal name in lambda instead of "_"

Reply:
it doesn't matter :) it's only short construction

Reply:
ok )